frontend/src/components/Map/index.js

```diff
@@ -341,76 +400,50 @@ export default function Map(props, { draggable = true }) {
     // add markers to map
     geojson.features.forEach(function (marker) {
       // create a HTML element for each feature
+      console.log(marker);
       var el = document.createElement("div");
       el.className = "marker";
       var myStory = marker.properties.myStory;
 
-      // check if mystory is null
-      if (!myStory) {
-        myStory = "";
-      }
-
-      var date = marker.properties.createdAt.substring(0, 10);
-      var popup = new mapboxgl.Popup({ offset: 25 }).setHTML(
-        "<h3>" + date + "</h3><p>" + myStory + "</p>"
-      );
-
-      map.on("mouseenter", "places", function (e) {
-        // Change the cursor style as a UI indicator.
-        map.getCanvas().style.cursor = "pointer";
-        var coordinates = e.features[0].geometry.coordinates.slice();
-        var description = e.features[0].properties.myStory;
-
-        // Ensure that if the map is zoomed out such that multiple
-        // copies of the feature are visible, the popup appears
-        // over the copy being pointed to.
-        while (Math.abs(e.lngLat.lng - coordinates[0]) > 180) {
-          coordinates[0] += e.lngLat.lng > coordinates[0] ? 360 : -360;
-        }
-
-        // Populate the popup and set its coordinates
-        // based on the feature found.
-        popup.setLngLat(coordinates).setHTML(description).addTo(map);
-      });
-
-      map.on("mouseleave", "places", function () {
-        map.getCanvas().style.cursor = "";
-        popup.remove();
-      });
+      content = "";
+      //add user story if has any
+      if (myStory)
```

Comment:
You need to edit the code in the master branch since in the master the we have filtered the story that is null or "".